utils/pathUtils.js

```diff
@@ -7,6 +7,20 @@ const fs = require("fs");
 const path = require("path");
 
 module.exports = {
+   /**
+    * @function destPath
+    * return true if the path exists.
+    * @param {string} destPath
+    * @return {Promise<boolean>}
+    */
+   checkPath: async (destPath) =>
+      await new Promise((resolve) => {
```

Comment:
Do we need `return` here ?

Reply:
No, because I don't put `{}` after `=>`.